tests/templates/kuttl/resources/13-assert.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 240
+skipLogOutput: true
+commands:
+  - script: kubectl get cm -n $NAMESPACE spark-resources-pod-template -o yaml | yq -r '.data."driver.yml"' | yq -r '.spec.containers[0].resources.limits.cpu' | grep 1500m
+  - script: kubectl get cm -n $NAMESPACE spark-resources-pod-template -o yaml | yq -r '.data."driver.yml"' | yq -r '.spec.containers[0].resources.requests.cpu' | grep 1
+  - script: kubectl get cm -n $NAMESPACE spark-resources-pod-template -o yaml | yq -r '.data."executor.yml"' | yq -r '.spec.containers[0].resources.limits.cpu' | grep 2
+  - script: kubectl get cm -n $NAMESPACE spark-resources-pod-template -o yaml | yq -r '.data."executor.yml"' | yq -r '.spec.containers[0].resources.requests.cpu' | grep 1
```

Comment:
Can this not be done declaratively ? Something like:
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: spark-resources-pod-template
spec:
  containers:
    - resources:
        - requests:
            cpu: 1
...
```

Reply:
hmmmm, das scheint leider nicht zu funktionieren, da das ganze Data-Element wird verglichen, inklusiv von UIDs:

```
         ...
        != actual: ---
        apiVersion: v1
        kind: Pod
        metadata:
          labels:
            app.kubernetes.io/instance: spark-resources
            app.kubernetes.io/managed-by: spark-k8s-operator
            app.kubernetes.io/name: spark-k8s
            app.kubernetes.io/version: "1.0"
          name: spark-driver
          ownerReferences:
            - apiVersion: spark.stackable.tech/v1alpha1
              kind: SparkApplication
              name: spark-resources
              uid: 6841c21a-50a2-4969-b529-665702180c26
        ...
```